sdk/cosmos/azure-spring-data-cosmos/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
-      <version>5.3.19</version> <!-- {x-version-update;org.springframework:spring-core;external_dependency} -->
+      <version>5.3.20</version> <!-- {x-version-update;org.springframework:spring-core;external_dependency} -->
```

Comment:
Now the version is managed here: https://github.com/Azure/azure-sdk-for-java/blob/e4b9532670db14c7e2ad9cd4d893f285909ea474/eng/versioning/external_dependencies.txt#L152

We should avoid update it manually.
Refs: https://github.com/Azure/azure-sdk-for-java/blob/e4b9532670db14c7e2ad9cd4d893f285909ea474/CONTRIBUTING.md#versions-and-versioning